keps/sig-node/20190604-cgroup-none.md

```diff
@@ -0,0 +1,174 @@
+---
+title: New cgroup driver: "none" (for Rootless)
+authors:
+  - "@AkihiroSuda"
+owning-sig: sig-node
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2019-06-04
+last-updated: 2019-06-04
+status: implementable
+---
+
+# New cgroup driver: "none" (for Rootless)
+
+## Table of Contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Future: cgroup2](#future-cgroup2)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Implementation Details](#implementation-details)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+- [Feature Gate](#feature-gate)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Summary
+
+Add "none" cgroup driver to kubelet so that kubelet (as well as CRI/OCI runtimes) can be executed without write permissions to cgroups.
```

Comment:
Is it a goal of "rootless" to remove all resource isolation?  If not, can we instead work towards the end-state of supporting cgroupv2 rather than disabling all resource isolation?